resources/hiveclients/java/hiveserver1-client/README.md

```diff
@@ -0,0 +1,26 @@
+#hiveserver1-client
```

Comment:
Maybe it is better to use something more "human", like:.

"Hive server version 1 client"